numpy/linalg/linalg.py

```diff
@@ -1615,17 +1615,20 @@ def svd(a, full_matrices=True, compute_uv=True, hermitian=False):
         # reversing is cheap by design in numpy, so we do so to be consistent
         if compute_uv:
             s, u = eigh(a)
-            s = s[..., ::-1]
-            u = u[..., ::-1]
-            # singular values are unsigned, move the sign into v
-            vt = transpose(u * sign(s)[..., None, :]).conjugate()
+            sgn = sign(s)
             s = abs(s)
+            sidx = s.shape[-1] - 1 - argsort(s[::-1])
```

Comment:
Needs a `...`

While correct ignoring that, this is slower than just `argsort(s)[..., ::-1]`, and we really don't care about how ties are resolved.